feature-pack/src/main/resources/configuration/host/host.xml

```diff
@@ -17,6 +17,11 @@
                 </authorization>
             </security-realm>
             <security-realm name="ApplicationRealm">
+                <server-identities>
+                    <ssl>
+                        <keystore path="application.keystore" relative-to="jboss.domain.config.dir" keystore-password="password" alias="server" key-password="password" generate-self-signed-certificate-host="localhost"/>
+                    </ssl>
+                </server-identities>
```

Comment:
This is fine but there are also host-master.xml and host-slave.xml in this folder.